drivers/sensor/ism330dhcx/ism330dhcx_trigger.c

```diff
@@ -37,7 +37,7 @@ static int ism330dhcx_enable_t_int(const struct device *dev, int enable)
 	}
 
 	/* set interrupt (TEMP DRDY interrupt is only on INT2) */
-	if (cfg->int_pin == 1)
+	if (cfg->int_gpio.pin == 1)
```

Comment:
I wonder how this works... `pin` can be an arbitrary number unless I miss something here

Reply:
It caught my eye as well, but as long as it doesn't change the logic of the driver we better postpone that fix for later?